internal/orchestrion/_integration/go.mod

```diff
@@ -63,8 +63,8 @@ require (
 	github.com/vektah/gqlparser/v2 v2.5.21
 	github.com/xlab/treeprint v1.2.0
 	go.mongodb.org/mongo-driver v1.17.1
-	google.golang.org/api v0.229.0
-	google.golang.org/grpc v1.72.0
+	google.golang.org/api v0.176.1
+	google.golang.org/grpc v1.71.1
```

Comment:
Importing `orchestrion` pushes the version up to `v1.71.1`, but we pin it at the end of the `go.mod`.